### DISCUSSION TITLE
FNF 0.3.3 version 1 (playtest)

### DISCUSSION THREAD

**Discussion by luckydog7** [luckydog7/Funkin-android#1955] [General]
new builds will be located here (source code and ios builds too):
https://github.com/FunkinDroidTeam/Funkin/releases/

This is a playtest and everything may change

Download:
https://github.com/FunkinDroidTeam/Funkin/releases/download/v0.3.3-playtest/Funkin.0.3.3.Playtest.apk

<hr /><em>This discussion was created from the release <a href='https://github.com/luckydog7/Funkin-android/releases/tag/v0.3.3.1'>FNF 0.3.3 version 1 (playtest)</a>.</em>

**Comment by Groovydilan** [comment]
I'm sorry, I know you've done a lot of work,and it would be rude to ask for something, but could you please return the custom control, since playing on the sound pad is not so easy.And I'm already used to playing with custom controls, again asking for myself is too selfish, so if there is the slightest opportunity, could you return it?

**Comment by Lucaspdk15** [comment]
Hi can you add the default arrow controls from the old versions? Because these touch screen controls make the game unplayable
![Screenshot_20240612-195918](https://github.com/luckydog7/Funkin-android/assets/155905778/1c7d25ea-0220-4ba8-8b12-0d75fcc238a4)

**Reply by Lucaspdk15** [reply]
Old controls is playble ![Screenshot_20240612-200330](https://github.com/luckydog7/Funkin-android/assets/155905778/fd41f3f6-1ba2-4b85-9686-056eeaf68ebd)
